coffee-backend/src/main/kotlin/chat/letscoffee/security/oauth2/OAuth2AuthenticationSuccessHandler.kt

```diff
@@ -0,0 +1,66 @@
+package chat.letscoffee.security.oauth2
+
+import chat.letscoffee.config.AppProperties
+import chat.letscoffee.exception.BadRequestException
+import chat.letscoffee.security.security.TokenProvider
+import chat.letscoffee.util.CookieUtils
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.core.Authentication
+import org.springframework.security.web.authentication.SimpleUrlAuthenticationSuccessHandler
+import org.springframework.stereotype.Component
+import org.springframework.web.util.UriComponentsBuilder
+import java.io.IOException
+import java.net.URI
+import javax.servlet.ServletException
+import javax.servlet.http.Cookie
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+@Component
+class OAuth2AuthenticationSuccessHandler @Autowired internal constructor(private val tokenProvider: TokenProvider, private val appProperties: AppProperties,
```

Comment:
No need for `@Autowired` here, all fields in constructor with @Component are automatically autowired